doc/reference/create-cardano.md

```diff
@@ -0,0 +1,177 @@
+# Create Cardano
+
+In [Shelley genesis](doc/reference/shelley-genesis.md) we discussed how to manually create a Shelley blockchain and how to create a testnet semi-automatically with `cardano-cli genesis create`. That does not cover what to do when we need a testnet that starts in Byron era and is updated to the latest era. This requires a fair amount of manual work: converting genesis and delegate keys from Byron to Shelley, creating genesis files, handling file hashing and manually updating the configuration file, etc.
```

Comment:
Suggestion: phrase the last part in the past tense
`does not cover` -> `did not cover`
`this requires` -> `this required`

to contrast with the newly introduced command